packager/mpd/base/xml/xml_node.cc

```diff
@@ -404,6 +408,77 @@ bool RepresentationXmlNode::AddVODOnlyInfo(const MediaInfo& media_info) {
   return true;
 }
 
+scoped_xml_ptr<xmlNode> RepresentationXmlNode::GetLiveOnlyInfo(
```

Comment:
`GetLiveOnlyInfo` doesn't quite seem like the right name for this, wouldn't something like GetSegmentTemplate be more clear about what it does?